js/modules/k6/ws/ws.go

```diff
@@ -247,17 +247,18 @@ func (mi *WS) Connect(url string, args ...goja.Value) (*WSHTTPResponse, error) {
 
 	if state.Options.SystemTags.Has(metrics.TagIP) && conn.RemoteAddr() != nil {
 		if ip, _, err := net.SplitHostPort(conn.RemoteAddr().String()); err == nil {
-			tags["ip"] = ip
+			// TODO: make as a not indexable
```

Comment:
I guess @na-- needs to weight in on what the API will be for that and whether this current API is good

Reply:
Also maybe have the same comment for the `url` as well?!?

Reply:
Forgot to reply here. The basic idea is this:
- we'll continue storing these non-indexable tags as a plain Go `map[string]string` in `lib.State.Tags`, i.e. a new field in the `lib.TagMap` struct currently called `metadata`, for historical reasons... :sweat_smile: 
- branching out will just copy it, though it should be `nil` most of the time, unless someone explicitly added a non-indexable tag through `vu.tags` in `k6/execution`
- per-testrun `--tags` and per-scenario `tags` are, by definition, indexable, since they will apply to all of the generated metrics, i.e. have super-low cardinality :sweat_smile: so they are always going to be kept just in atlas, non-indexable tags only come to play on the VU level
- we know which system tags will be non-indexable (url, error, vu, iter) so, if they are enabled, just add their values in that map instead of in atlas
- we also add any per-measurement (i.e. per HTTP request, per `add()`, etc.) custom non-indexable metrics to the map instead of the atlas node

After this PR stabilizes a bit, I'll try to rebase my changes on top of it and make another PR to show what I mean.

Reply:
I guess we should resolve this @na--?

Reply:
And also remove the comment?